asciidoc/courses/gds-product-introduction/modules/2-graph-management/lessons/3-c-native-projection/lesson.adoc

```diff
@@ -12,6 +13,6 @@ include::./questions/verify.adoc[leveloffset=+1]
 [.summary]
 == Lesson Summary
 
-In this lesson we went over native projections, the primary graph projection mechanism in GDS.  Native Projections are optimized for efficiency and performance to support graph data science at scale.
+In this lesson we went over native projections, the easiest graph projection mechanism in GDS.
```

Comment:
We should avoid describing this as "easy" (as per Stefano's workshop)

Reply:
I wanted to express `ease of use` as used at https://neo4j.com/docs/graph-data-science/current/management-ops/graph-creation/. 

`simplest` is not any better is it?

Any other phrasing ideas?

Reply:
Easiest can be demoralizing. What we consider "easy" someone might consider hard. I have changed to quickest.